runtime-acceptance-tests/src/main/resources/config-env.properties

```diff
@@ -1,7 +1,9 @@
 realm=activiti
-auth.url=http://${SSO_HOST}/auth/realms/${realm}/protocol/openid-connect/token
-audit.event.url=http://${GATEWAY_HOST}/audit
-runtime.bundle.url=http://${GATEWAY_HOST}/rb-my-app
-query.url=http://${GATEWAY_HOST}/query
-graphql.ws.url=ws://${GATEWAY_HOST}/ws/graphql
-graphql.url=http://${GATEWAY_HOST}/graphql
\ No newline at end of file
+sso.url=${sso.protocol:http}://${sso.host}/auth
+auth.url=${sso.url}/realms/${realm}/protocol/openid-connect/token
+gateway.url=${gateway.protocol:http}://${gateway.host}
+audit.event.url=${gateway.url}/audit
+runtime.bundle.url=${gateway.url}/rb-my-app
+query.url=${gateway.url}/query
+graphql.ws.url=#{"${gateway.url}".replace('http', 'ws')}/ws/graphql
```

Comment:
@mteodori For `https` in the GATEWAY_HOST, the graphql.ws.url should use `wss:` protocol scheme..

Reply:
that is what the replacement is doing, I think I should just change to replaceFirst

Reply:
all good with https disabling the non working tests and security policies

Reply:
@mteodori I have HTTPS in my environment, I will try deploy and run acceptance tests there with your change and then check notes with you.